vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -1093,7 +1093,6 @@ testWasmRuntimesInstalled() {
   local test="testWasmRuntimesInstalled"
   local wasm_runtimes_path=${1}
   local shim_version=${2}
-  shim_version="v${shim_version}"
```

Comment:
we're sure this is correct?

Reply:
Actually no. I just found that this line can't be removed, although it's another bug related to this line. I will add it back for now.